tests/acceptance/acceptance_test.go

```diff
@@ -251,25 +251,71 @@ var _ = Describe("Acceptance", func() {
 					profilesv1.ProfileDescription{
 						Name:        "nginx-2",
 						Description: "nginx 1",
-						Catalog:     catalogName,
+						Catalog:     sourceName,
 					},
 				))
 			})
 		})
 
+		getProfile := func(profileName string) (profilesv1.ProfileDescription, error) {
```

Comment:
Given that this is only used in a test, it doesn't need to return an error, it can just `Expect` things and fail.

Reply:
Also, it could return a pointer to a ProfileDescription?

Reply:
>Given that this is only used in a test, it doesn't need to return an error, it can just Expect things and fail.

There are multiple callers of this function, one of which expects a specific type of error to have occurred,, so moving the assertions to this function will not let us do that.

Reply:
You could pass that in as a param?

Reply:
There are multiple code paths in this function that can return an error. Which of those calls will that param be checked against? The `(value, error)` return type is a well-established pattern. Moreover, it's more verbose to replace `return err` calls with assertions.